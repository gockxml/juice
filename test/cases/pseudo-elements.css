a {
  text-decoration: underline;
}

a:before,
a:after {
  content: "a";
}

* a:before {
  content: 'Â®\+';
}

a:after,
a::after {
  font-weight: bold;
}

a:first-child {
  color: blue;
}

b:after {
 content: " ";
}

b:after {
 content: "b";
}
