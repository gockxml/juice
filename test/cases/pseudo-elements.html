<html><body>
<a href="#">Test</a>
<b></b>
<b></b>
</body></html>
