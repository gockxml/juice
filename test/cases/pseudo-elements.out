<html><body>
<a href="#" style="text-decoration: underline; color: blue;"><span>Â®+</span>Test<span style="font-weight: bold;">a</span></a>
<b><span>b</span></b>
<b><span>b</span></b>
</body></html>
